diff/fd/crosslaplacian.go

```diff
@@ -0,0 +1,190 @@
+// Copyright ©2017 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package fd
+
+import (
+	"math"
+	"sync"
+)
+
+// CrossLaplacian computes a Laplacian-like quantity for a function of two vectors
+// at the locations x and y.
+// It computes
+//  ∇_y · ∇_x f(x,y) = \sum_i ∂^2 f(x,y)/∂x_i ∂y_i
```

Comment:
Do you have a reference for this, where this comes up, what is it used for? For example, I think that the restriction len(x)=len(y) could be dropped if
`∇_y · ∇_x f(x,y) = \sum_i \sum_j ∂^2 f(x,y)/∂x_i ∂y_j`